app/derivative-services/jp2_derivative_service.rb

```diff
@@ -180,4 +180,17 @@ def cleanup_derivative_metadata(derivatives:)
         buffered_persister.save(change_set: updated_change_set)
       end
     end
+
+    def storage_adapter
```

Comment:
Any reason you moved this?

Reply:
A bit of OCD. Putting methods in alphabetical order.

Reply:
Change these back?

Reply:
Nah, it's fine.